source/_patterns/00-atoms/components/list~teaser-content.json

```diff
@@ -0,0 +1,9 @@
+{
+  "isOrdered": false,
+  "prefix": "bullet",
+  "classes": "list--teaser",
+  "items": [
+    "item 1",
```

Comment:
As the pattern variant addresses items containing links, please include a link in one of these item examples.